docs/Salesforce-connector.md

```diff
@@ -0,0 +1,32 @@
+# Salesforce Connection
+
+Description
+-----------
+Use this connection to access data in Salesforce.
+
+Properties
+----------
+**Name:** Name of the connection. Connection names must be unique in a namespace.
+
+**Description:** Description of the connection.
+
+**Username:** Salesforce username.
+
+**Password:** Salesforce password.
+
+**Security Token:** Salesforce security Token. If the password does not contain the security token the plugin
```

Comment:
Can we add a comma "If the password does not contain a security token, "

Reply:
Done